.github/workflows/release.yml

```diff
@@ -196,19 +196,19 @@ jobs:
   pypi-release:
     name: Pypi release
     # only release to PyPi if we're not testing - will release to PyPi test when workflow gets rewritten
-    if: inputs.test_run  == 'false'
+    if: ${{ inputs.test_run  == false }}
```

Comment:
The curley braces are not needed per [the docs](https://docs.github.com/en/actions/learn-github-actions/expressions#:~:text=When%20you%20use%20expressions%20in%20an%20if%20conditional%2C%20you%20may%20omit%20the%20expression%20syntax%20(%24%7B%7B%20%7D%7D)%20because%20GitHub%20automatically%20evaluates%20the%20if%20conditional%20as%20an%20expression.).  You just need to remove the single quotes.  There's some context weirdness documented [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs:~:text=Note%3A%20The%20workflow%20will%20also%20receive%20the%20inputs%20in%20the%20github.event.inputs%20context.%20The%20information%20in%20the%20inputs%20context%20and%20github.event.inputs%20context%20is%20identical%20except%20that%20the%20inputs%20context%20preserves%20Boolean%20values%20as%20Booleans%20instead%20of%20converting%20them%20to%20strings.) with `github.event.inputs` vs just `inputs`.

Reply:
ahh that makes sense, but I wonder if we should use them anyways since that's more consistent with GHA syntax?